src/java.base/share/classes/java/io/Console.java

```diff
@@ -82,320 +80,251 @@
  * manually zero the returned character array after processing to minimize the
  * lifetime of sensitive data in memory.
  *
- * <blockquote><pre>{@code
+ * {@snippet lang=java :
  * Console cons;
  * char[] passwd;
  * if ((cons = System.console()) != null &&
  *     (passwd = cons.readPassword("[%s]", "Password:")) != null) {
  *     ...
  *     java.util.Arrays.fill(passwd, ' ');
  * }
- * }</pre></blockquote>
+ * }
  *
  * @author  Xueming Shen
  * @since   1.6
+ * @sealedGraph
```

Comment:
I'm not sure about sealedGraph because the there is nothing for user code to matching on.

Reply:
OK, removed